### ISSUE TITLE
Deblurring Task

### ISSUE THREAD

**Issue by ozanpkr** [JingyunLiang/VRT#31]
I have a basic question about memory management.
When I tried deblurring task on my dataset I get this error:

`tcmalloc: large alloc 4556390400 bytes == 0x7fdce32c8000 @  0x7fdf59255b6b 0x7fdf59275379 0x7fde8e8b750e 0x7fde8e8a97c2 0x7fdec8c642d8 0x7fdec8c64de7 0x7fdec9375928 0x7fdec8d08b07 0x7fdec8d09415 0x7fdec93dee82 0x7fdec92b1a2d 0x7fdec8d142e1 0x7fdec94dc622 0x7fdec8fed79a 0x7fdec8d101da 0x7fdec94dd5b2 0x7fdec90b17c2 0x7fdeca0d677a 0x7fdeca0d6d65 0x7fdec90fd35d 0x7fdf43d5f2e0 0x593835 0x548c51 0x5127f1 0x549e0e 0x593fce 0x548ae9 0x51566f 0x4bc98a 0x533274 0x4d3969
tcmalloc: large alloc 4556390400 bytes == 0x5e4e2000 @  0x7fdf59255b6b 0x7fdf59275379 0x7fde8e8b750e 0x7fde8e8a97c2 0x7fdec881b10f 0x7fdec881ba51 0x7fdec881baa4 0x7fdec8cee9ae 0x7fdec93dbaea 0x7fdec919d38c 0x7fdec93c262f 0x7fdec91d81e0 0x7fdf43fa7371 0x7fdec8cf2cf0 0x7fdec9573994 0x7fdec8fc2467 0x7fdec93c1995 0x7fdec9005a59 0x7fdf43cf35e1 0x593784 0x548c51 0x51566f 0x593dd7 0x5118f8 0x593dd7 0x5118f8 0x549576 0x604173 0x5f5506 0x5f8c6c 0x5f9206
/usr/local/lib/python3.7/dist-packages/torch/utils/checkpoint.py:25: UserWarning: None of the inputs have requires_grad=True. Gradients will be Nonewarnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
`
I reduced to 128x128 but nothing has changed.Could you give any suggestion ?

**Comment by JingyunLiang** [comment]
You can reduce the video size by using (e.g., 6x64x64) or use machines with larger memory.
